docs/cvl/cvl4/changes.md

```diff
@@ -0,0 +1,371 @@
+Syntax changes introduced in CVL 4
```

Comment:
Customers don't know about CVL1,2,3 so the number 4 is much too surprising.
Prover majors are not correlated necessarily with CVL versions. 
While a breaking-change or otherwise significant update of CVL means we'll bump the Prover version, it's not symmetric: We could have a major Prover version with 0 CVL updates.